### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by FrancisJMR**
New Researchers ID & Access Claims landing page.  flow_of_claims.png still needs to be added.

### CODE REVIEW

File: researcher_ids/README.md
Comment (cdvoisin) [inline researcher_ids/README.md]: Replace "ne" with "one"?

File: researcher_ids/README.md
Comment (cdvoisin) [inline researcher_ids/README.md]: The PNG file is missing?

**Review by cdvoisin — COMMENTED** [review]
Thanks for putting this together. How about we land it with the current links and then update the links later to point to the github docs instead of the Google docs.

File: researcher_ids/README.md
Comment (cdvoisin) [reply researcher_ids/README.md]: Also it may be worth putting image assets in the /assets/img/ directory.

File: researcher_ids/README.md
Comment (FrancisJMR) [reply researcher_ids/README.md]: Fixed in new commit.

File: researcher_ids/README.md
Comment (FrancisJMR) [reply researcher_ids/README.md]: I've changed the path to /assets/img/  --- I'd don't have push access to the repository to add the image in that folder however. Can you try it? The image should be named "flow_of_claims.png"

**Review by cdvoisin — APPROVED** [review]

**Comment by FrancisJMR** [comment]
Ok I've added the png file and incorporated the changes to the introduction from Tommi and I found in the Google Doc.  I cannot merge this back to master but it should be good to go.

File: researcher_ids/README.md
Comment (cdvoisin) [inline researcher_ids/README.md]: Remove: "in the virtual world that travel with the researcher across"
Remove: "sharing" (after the words "federated data")

**Pull request merged**
